src/cogs/general/general_rickrollscanner.py

```diff
@@ -0,0 +1,58 @@
+"""
+Uses get request to check if the video is a rick roll using keywords on youtube.
+"""
+import logging
+import requests
+from discord.ext import commands
+
+
+logger = logging.getLogger(__name__)
+
+
+class RickRollScanner(commands.Cog):
+    """
+    Uses youtube get request to check if the video is a rick roll using keywords.
+    """
+
+    def __init__(self, bot):
+        self.bot = bot
+
+    @commands.slash_command(description="Check for potential rick roll by video id")
+    async def imbored(self, ctx, prompt):
+        """
+        Send a get request to youtube to check if the video is a rick roll
+        """
+        logger.info("%s used the %s command."
+                    , ctx.author.name
+                    , ctx.command)
+        gen_url = f"https://www.youtube.com/watch?v={prompt}"
+        data = requests.get(
+            gen_url, timeout=5).text
+        
+        forbidden_keywords = [
```

Comment:
dangerous game. What if I send a video called "**rick** ....  and morty" or "Learn Python, dont give **up**!"

Reply:
I see, so it's only a warning. Should be fine if we just change the command name. Could also add it to the moderation cog so that when a youtube link is detected, we auto scan it and send the warning.